tests/storage/hotplug.go

```diff
@@ -425,6 +425,29 @@ var _ = Describe(SIG("Hotplug", func() {
 		return vm
 	}
 
+	createBootableHotplugVM := func(storageClass string) *v1.VirtualMachine {
+		opts := []libvmi.Option{
+			libvmi.WithCloudInitNoCloud(libvmifact.WithDummyCloudForFastBoot()),
+		}
+		dv := libdv.NewDataVolume(
+			libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
```

Comment:
Use Alpine instead?

Reply:
All the tests in this file are using cirros. So I think it would be best to be consistent unless there's a really good reason to go with alpine

Reply:
Should be a follow up too. In mid to long term we should move away from cirros though (in the whole tests codebase).